test/CadlRanchProjectsNonAzure.Tests/type-property-nullable.cs

```diff
@@ -0,0 +1,248 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Xml;
+using Scm._Type.Property.Nullable;
+using Scm._Type.Property.Nullable.Models;
+using AutoRest.TestServer.Tests.Infrastructure;
+using Azure.Core;
+using NUnit.Framework;
+using System.ClientModel;
+
+namespace CadlRanchProjectsNonAzure.Tests
+{
+    public class TypePropertyNullableTests : CadlRanchNonAzureTestBase
+    {
+        [Test]
+        public Task Type_Property_Nullable_String_getNonNull() => Test(async (host) =>
+        {
+            var response = await new NullableClient(host, null).GetStringClient().GetNonNullAsync();
+            Assert.AreEqual("foo", response.Value.RequiredProperty);
+            Assert.AreEqual("hello", response.Value.NullableProperty);
+        });
+
+        [Test]
+        public Task Type_Property_Nullable_String_getNull() => Test(async (host) =>
+        {
+            var response = await new NullableClient(host, null).GetStringClient().GetNullAsync();
+            Assert.AreEqual("foo", response.Value.RequiredProperty);
+            Assert.AreEqual(null, response.Value.NullableProperty);
+        });
+
+        [Test]
+        public Task Type_Property_Nullable_String_patchNonNull() => Test(async (host) =>
+        {
+            var value = new
+            {
+                requiredProperty = "foo",
+                nullableProperty = "hello"
+            };
+            var response = await new NullableClient(host, null).GetStringClient().PatchNonNullAsync(BinaryContent.Create(BinaryData.FromObjectAsJson(value)));
```

Comment:
I wonder why this operation (or maybe all the operations) does not have a convenience method.
Could you please investigate?
If possible we should use convenience methods.

Reply:
I see - we did not handle patch operations, therefore this does not have a convenience method.
Never mind then